src/test/java/com/training/apitests/ApiTests.java

```diff
@@ -0,0 +1,100 @@
+package com.training.apitests;
+
+import com.training.utilities.ApiUtilities;
+import org.junit.jupiter.api.Test;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.simple.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import java.io.IOException;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+
+public class ApiTests {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    ApiUtilities apiUtilities;
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/getIdResponse.csv")
+    public void getIdResponse(String name,String id) {
+        RestAssured.baseURI = "https://api.thecatapi.com/v1/breeds";
+        RequestSpecification httpRequest = RestAssured.given();
+        httpRequest.header("x-api-key", "37b0987e-eb13-4193-82b1-56f33b574ac0");
+        Response response = httpRequest.get();
+
+        Assertions.assertEquals(response.getStatusCode(),200);
+
+        List<String> breedName = response.jsonPath().getList("name");
+        List<String> breedId = response.jsonPath().getList("id");
+        logger.info(breedId.get(breedName.indexOf(name)));
+
+        Assertions.assertEquals(breedId.get(breedName.indexOf(name)),id);
+    }
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/getWikipediaUrl.csv")
+    public void getWikipediaUrl(String id, String url){
+        RestAssured.baseURI = "https://api.thecatapi.com/v1/breeds";
```

Comment:
lets make the baseURI as -> https://api.thecatapi.com and also lets add this to json file